dev-project/docker-compose.yml

```diff
@@ -32,7 +32,13 @@ services:
 
     volumes:
       - ../:/usr/src/app
-      # - /usr/src/app/.virtualenvs/
+      # Do not mount .virtualenvs dirs from host, but recreate it in the container as empty dirs:
+      #   - Makes install script faster in the container because it doesn't need to "jump" back
+      #     to host every time when installing that many files to dev-project/.virtualenvs/
+      #   - Doesn't create container specific dev-project/.virtualenvs/ files on host
+      #   - Doesn't bring host specific .virtualenvs/ files to container
+      - /usr/src/app/dev-project/.virtualenvs/
+      - /usr/src/app/.virtualenvs
```

Comment:
Should we be mounting - /usr/src/app/.virtualenvs ?
All virtualenvs created by the dev-project docker should go into - /usr/src/app/dev-project/.virtualenvs/

Reply:
removed `/usr/src/app/.virtualenvs` from volumes list by  https://github.com/transferwise/pipelinewise/pull/206/commits/90de622995051b67f97f7b8b7330bc8808f06aff